tests/lib/hardware/hardware_base.py

```diff
@@ -120,12 +120,15 @@ def node_remove(self, node: NodeBase):
     def boot_nodes(self, masters: int, workers: int, offset: int = 0):
         logger.info("boot nodes")
 
-    def prepare_nodes(self, limit_to_nodes: List[NodeBase] = []):
+    def prepare_nodes(self, limit_to_nodes: List[NodeBase] = [],
+                      extra_vars=settings.ANSIBLE_EXTRA_VARS):
```

Comment:
I don't think the extra_vars should be limited to prepare_nodes. As it is in the config at the moment, we should just apply it to every ansible run.

Reply:
Actually I used the extra_vars in all run, but only defaulted it in prepare.
I updated to always default to settings.ANSIBLE_EXTRA_VARS directly in ansible_run_playbook()